tools/db_crashtest.py

```diff
@@ -970,10 +974,6 @@ def finalize_and_sanitize(src_params):
             # we have a fix that allows auto recovery.
             dest_params["exclude_wal_from_write_fault_injection"] = 1
             dest_params["metadata_write_fault_one_in"] = 0
-    if dest_params.get("disable_wal") == 1:
```

Comment:
I noticed we had 2 checks for `disable_wal` and consolidated the parameter sanitization